Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -1048,4 +1049,24 @@ BOOST_PYTHON_MODULE(rdMolDescriptors) {
   python::def("CalcNumBridgeheadAtoms", numBridgeheadAtoms,
               (python::arg("mol"), python::arg("atoms") = python::object()),
               docString.c_str());
+
```

Comment:
Re: "good path"
I certainly think it makes sense.

Re: python calculators
That would be useful and it's relatively easy to handle with boost. I'm planning on doing an example of this for aromaticity perception sometime in the next couple days. That could maybe serve as a model?